src/Forms/Form/Form.stories.mdx

```diff
@@ -56,11 +56,313 @@ const fields = useMemo(
+
+#### How to use in a form?
+
+```ts
+const fields = useMemo(
+	() =>
+		[
+			{
+				name: "textEditor",
+				label: "Text Editor Label",
+				type: "textEditor",
+				required: true,
+				inputSettings: {
+					spellcheck: true,
+					direction: "rlt",
+					language: "en",
+					maxCharacters: 20,
+				},
+				disabled: false,
+				helperText: "Helper text",
+				instructionText: "Instruction text"
+			},
+		] as FieldDef<TextEditorDef>[],
+	[]
+);
+```
+
+### FormFieldTable
+- Tables are used in some cases inside the form after selecting a list of elements or by using arrays
+- The data table component allows for customization with additional functionality, as needed by your product’s users.
+- **Playground**: https://simpleviewinc.github.io/sv-mosaic/?path=/story/formfields-formfieldtable--playground
+
+#### Props (Input Settings)
+
+* **handleAddElement** - `() => void` required - Function used to create a new row in the table.
+* **handleEdit** - `(rowIndex: number) => void` required - Function that defines how the clicked row should be editted.
+* **handleDelete** - `() => void` required - Function that will be executed when clicking on the trash icon. It can be used to show some kind of confirmation before removing the row.
+* **extraActions** - `array` of `object` optional - Possible actions that the table could execute and display.
+	* **label** - `string` - Describes the action.
+	* **icon** - `MosaicMIcon` - MUI icon.
+	* **actionFnc** - `actionFnc: (rowIndex: number) => void;` - Function that is executed when the corresponding icon is clicked.
+* **headers** - `string[]` required - Array of the table headers.
+
+#### How to use in a form?
+
+```ts
+const possibleTableRows = [
+	{
+		id: "1",
+		items: ["John", "john@email.com", "01/01/2021", "3231-962-7516"],
+	},
+	{
+		id: "2",
+		items: ["Sally", "sally@email.com", "12/24/2020", "011-962-111"],
+	},
+	{
+		id: "3",
+		items: ["Maria", "maria@email.com", "12/01/2020", "788-962-7516"],
+	}
+];
+
+const addTableRow = (): void => {
+	const tableDataLength = dataState.table ? dataState.table.length : 0;
+
+	if (tableDataLength === 0) {
+		dispatch(
+			actions.setFieldValue({
+				name: "table"
+				value: [possibleTableRows[0]],
+			})
+		);
+	} else if (tableDataLength >= possibleTableRows.length) {
+		alert("There are no more elements to add");
+	} else {
+		let index;
+
+		for (let i = 0; i < possibleTableRows.length; i++) {
+			const element = possibleTableRows[i];
+			const isRepeatedRow = dataState.table.includes(element);
+			if (!isRepeatedRow) {
+				index = i;
+				break;
+			}
+		}
+
+		dispatch(
+			actions.setFieldValue({
+				name,
+				value: [...dataState.table, possibleTableRows[index]],
+			})
+		);
+	}
+};
+
+const fields = useMemo(
+	() =>
+		[
+			{
+				disabled: false,
+				helperText: 'Helper text',
+				instructionText: 'Instruction text',
+				inputSettings: {
+					handleAddElement: addTableRow,

```

Comment:
If we change this to just handleAddElement: myAddFuncion, then by reading this and the prop list I think it's more than enough for devs to decide how the function should work like.